### ISSUE TITLE
Verification of signed message fails if digest formatted with newlines

### ISSUE THREAD

**Issue by Fredbasset** [XML-Security/signxml#155]
The digest for signing and verification is currently formatted as a single string of base64 encoded characters with no line breaks. When receiving a message from a client which formats the base64 digest with line breaks verification fails because the comparison is done using a textual comparison of the encoded data, rather than a comparison of the raw bytes.

Converting the base64 string to the raw bytes before comparison would solve the issue, and correctly verify documents where the digest formatting does not exactly match the `signxml` choices.

in __init__.py, XMLVerifier.verify

```python
if digest_value.text != self._get_digest(payload_c14n, self._get_digest_method(digest_algorithm)):
    raise InvalidDigest("Digest mismatch for reference {}".format(len(verify_results)))
```
-->
```python
if b64decode(digest_value.text) != b64decode(self._get_digest(payload_c14n, self._get_digest_method(digest_algorithm))):
    raise InvalidDigest("Digest mismatch for reference {}".format(len(verify_results)))
```

**Comment by kislyuk** [comment]
Released a fix in v2.8.0, please test.

**Comment by Fredbasset** [comment]
Looks good to me!
